Information as an Asset Through Business Intelligence and Data Analytics (#89)
ippontech/blog-usa: posts/Information-as-an-Asset-Through-Business-Intelligence-and-Data-Analytics.md

> ---

Reviewer: Can you rename the file to lowercase?

--> `information-as-an-asset-through-business-intelligence-and-data-analytics.md`

authors:
- Rita Barlow
tags:
- Data Analytics
- Business Intelligence
- Data 
- Big Data 
- Information 
date: 2019-02-05T12:21:50.000Z
title: "Information as an Asset Through Business Intelligence and Data Analytics"

> image: BI_VS_Data_Analytics.png

Reviewer: The `image` tag in the metadata is to display the image at the header image. I see you also included the image in the post, so you may want to keep only one of these. If you keep the header image, the full link will be needed.

---
## Make Better Decisions Faster

Business Intelligence and Data Analytics are critical practices to ensure your organization is running efficiently and proactively. In order to determine _how_ these fundamentals of data practices can help improve operations within your organization, we must first be able to determine the difference between the two, and answer how they can be utilized moving forward.

> **Business Intelligence (BI)** refers to the technologies, applications, and practices used to collect, integrate, analyze, and visualize your organization&#39;s data as valuable information.  This technology-driven process is used to help executives, managers, and other data consumers in your organization make informed business decisions. BI allows your organization to identify business opportunities, gauge productivity, and optimize spending for areas like sales and marketing by predicting customer behavior and analyzing trends over time.

Reviewer: ```suggestion
**Business Intelligence (BI)** refers to the technologies, applications, and practices used to collect, integrate, analyze, and visualize your organization's data as valuable information.  This technology-driven process is used to help executives, managers, and other data consumers in your organization make informed business decisions. BI allows your organization to identify business opportunities, gauge productivity, and optimize spending for areas like sales and marketing by predicting customer behavior and analyzing trends over time.
```

**Data Analytics (DA)** refers to the act of converting unstructured or raw data into a comprehensive or informational format that can be used to support your organization at all levels. Your organization will be able to make quality decisions, improve planning and forecasting, enable consistent data across your enterprise, improve speed, accessibility, ease of use, and ultimately lower costs and increase operational efficiency. The benefits to how Data Analytics can improve your organization are nearly endless. The following are a couple examples where Data Analytics can be applied to your data to make it a powerful informational asset.

> - **Customer Experience:** Data Analytics allows you to get to know your customer by compiling and understanding data regarding their habits, preferences, and needs. This enables your organization to tailor offerings and solutions that will optimize your customers experience and outcomes, making them more likely to return to your organization in the future. This could be applied to product offerings, services, geographical locations of stores, frequency of contact, medium of contact, etc.

Reviewer: ```suggestion
- **Customer Experience:** Data Analytics allows you to get to know your customers by compiling and understanding data regarding their habits, preferences, and needs. This enables your organization to tailor offerings and solutions that will optimize your customers' experience and outcomes, making them more likely to return to your organization in the future. This could be applied to product offerings, services, geographical locations of stores, frequency of contact, medium of contact, etc.
```

- **Risk Mitigation:** Information becomes a tool for predictability and forecasting. Your organization can improve risk mitigation and fraud through the use of statistical, network, path, and big data methodologies for predictive behavioral models that alert your organization and allow for real-time responses or proactive mitigation to threats.

![](https://github.com/ritabarlow/blog-usa/blob/master/images/2019/01/BI_VS_Data_Analytics.png) 

> Both of these practices use data as information to help you make decisions with slight subtleties. Through the use of BI, leaders in your organization can leverage existing data and present it in a way that allows your organization to learn from past mistakes and build on past successes. Replicating what works and changing what has not. As a subset of Business Intelligence, Data Analytics allows your organization to be truly intelligent by anticipating developments, driving innovation, and accurately making predictions that will help your organization in the future. To fully utilize data as an asset in your organization you need both Business Intelligence and Data Analytics working together.

Reviewer: ```suggestion
Both of these practices use data as information to help you make decisions with slight subtleties. Through the use of BI, leaders in your organization can leverage existing data and present it in a way that allows your organization to learn from past mistakes and build on past successes. Replicating what works and changing what has not. As a subset of Business Intelligence, Data Analytics allows your organization to be truly intelligent by anticipating developments, driving innovation, and accurately making predictions that will help your organization in the future. To fully utilize data as an asset in your organization, you need both Business Intelligence and Data Analytics working together.
```
